ledger/internal/applications.go

```diff
@@ -202,7 +202,7 @@ func (cs *roundCowState) kvDel(key string) error {
 	return nil
 }
 
-func (cs *roundCowState) NewBox(appIdx basics.AppIndex, key string, size uint64, appAddr basics.Address) error {
+func (cs *roundCowState) NewBox(appIdx basics.AppIndex, key string, value string, appAddr basics.Address) error {
```

Comment:
nit: `size uint64` to `value string`, is it because of convenience `box_put` to create with default value?

Reply:
Yes, since `box_put` can create a box with an initial value, we need to produce the byte slice on the caller side now, instead of passing the length in and allocating on the callee side.